hack/update-codegen.sh

```diff
@@ -588,6 +588,10 @@ function codegen::applyconfigs() {
         done
     fi
 
+    git_find -z ':(glob)staging/src/k8s.io/client-go/**/*.go' \
+        | { xargs -0 grep -l -s --null "^// Code generated by applyconfiguration-gen. DO NOT EDIT.$" || true; } \
```

Comment:
I....it never occured to me that it would take the same inputs, but of course it does.  Except flags.

`git ls-files` takes `-cmo` for cached, modified, other.  `git grep` takes `--cached --untracked`

Let me see what I can do.

Reply:
fixed in next push